### ISSUE TITLE
support for new operators?

### ISSUE THREAD

**Issue by brightening-eyes** [ggerganov/ggml#390]
hello.
beside add, sub, mul, div, mul_mat, norm, etc, can batch normalization, layer normalization, lstm, gru, etc be added in order to add other types of models like 1d text cnns or res nets?
should they be implemented in my code with custom operators?
or, how would I convert them into ggml's format from pytorch?
thanks.
